grafana/mattermost-notification-metrics.json

```
@@ -76,6 +76,7 @@
       }
     ]
   },
+  "description": "A dashboard displaying notification health metrics, and distributions around notifications that were not sent, both deliberate and indeliberate. \n\nThe dashboard is broken in half by Websocket events (POSTED events from the server to web clients) and Push events (events sent to the push proxy to be received by mobile clients).",
```

nit: minor suggestion to change `broken` as at first read I misinterpreted as it was not working

```suggestion
  "description": "A dashboard displaying notification health metrics, and distributions around notifications that were not sent, both deliberate and indeliberate. \n\nThe dashboard is split in half by Websocket events (POSTED events from the server to web clients) and Push events (events sent to the push proxy to be received by mobile clients).",
```